libs/image/include/image/KtxBundle.h

```diff
@@ -106,7 +106,7 @@ class KtxBundle {
     /**
      * Parses the key="sh" metadata and returns 3 bands of data.
      *
-     * Assumes 3 bands for a total of 9 coefficients.
+     * Assumes 9 bands for a total of 27 coefficients.
```

Comment:
The comment is wrong. 
It was correct before except that 1 coefficient is 3 float values (RGB).
So it's *3* bands, and 27 floats (3^2  RGB coefficients).